src/store/actions.js

```diff
@@ -95,6 +95,10 @@ const setGasPrice = function({ commit }, gasPrice) {
   commit('SET_GAS_PRICE', gasPrice);
 };
 
+const setMinimumGasPrice = function({ commit }, minimumGasPrice) {
+  commit('SET_MINIMUM_GAS_PRICE', minimumGasPrice);
+};
+
```

Comment:
If the minimum gas price is located on the network object then there is not reason to also add a new property to the store/state.